oap-ws-openapi-ws/src/test/resources/META-INF/oap-module.conf

```diff
@@ -0,0 +1,8 @@
+name = oap-ws-openapi-ws-test
+dependsOn = oap-ws-openapi-ws
+services {
+  test-service {
```

Comment:
usually we are not specifying test classes as oap services. Check `oap.ws.api.ApiWSTest` for an example

Reply:
I used oap-ws-api module [configuration ](https://github.com/oaplatform/oap-ws/blob/master/oap-ws-api-ws/src/test/resources/META-INF/oap-module.conf) as an example for testing.

Reply:
I see now, it is just web service. Do you want to add Test for `TestWS`? Ot the aim was just to use it for docs?

Reply:
Yes, the main goal is to describe `TestWS` web service via OpenApi. There are no specific logic in this class which should be covered with tests.